kodi_addon_checker/__main__.py

```diff
@@ -39,15 +40,47 @@ def main():
                                      the current directory.")
     parser.add_argument("--version", action="version",
                         version="%(prog)s 0.0.1")
-    parser.add_argument("add_on", metavar="add-on", type=dir_type, nargs="*",
-                        help="optional add-on directories")
+    parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
     ConfigManager.fill_cmd_args(parser)
     args = parser.parse_args()
 
-    repo_path = os.path.abspath(os.getcwd())
-    config = Config(repo_path, args)
-    ConfigManager.process_config(config)
-    check_repo(config, repo_path, args.add_on)
+    current_dir = os.path.abspath(os.getcwd())
```

Comment:
If we pass the directory to the checker, this folder might be completly unrelated. Therefore I think we should never use this in that case.